node/src/distributions/mainnet.rs

```diff
@@ -42,6 +45,36 @@ fn read_contents_to_substrate_accounts(path_str: &str) -> BTreeMap<AccountId, f6
 	accounts_map
 }
 
+fn read_investor_accounts_to_multiaddress(path_str: &str) -> BTreeMap<MultiAddress, f64> {
+	let mut path = get_git_root();
+	path.push(path_str);
+	let json = read_contents(&path);
+	let json_obj = json.as_object().expect("should be an object");
+	let mut accounts_map = BTreeMap::new();
+	for (key, value) in json_obj {
+		// eth address start with `0x`
+		let first = key.chars().nth(0).unwrap();
+		if first == '0' {
```

Comment:
make use of https://doc.rust-lang.org/stable/std/primitive.str.html#method.starts_with to be extra sure, since maybe substrate address also starts with 0